tests/puppeteer/tools/linkchecker.js

```diff
@@ -0,0 +1,158 @@
+const puppeteer = require('puppeteer');
+const {expect} = require('chai');
+const fs = require('fs');
+const urlsList = require('./urls.js');
+
+const reportPath = 'tools/reports';
+// filename
+const curDate = new Date();
+const dateString = curDate.toJSON().slice(0, 10);
+const hours = curDate.getHours();
+const minutes = curDate.getMinutes();
+const seconds = curDate.getSeconds();
+const filename = `report_${dateString}_${hours}-${minutes}-${seconds}`;
+
+
+const URL_FO = process.env.URL_FO || 'http://localhost/prestashop/';
+const URL_BO = process.env.URL_BO || `${URL_FO}admin-dev/`;
+const LOG_PASSED = process.env.LOG_PASSED || false;
+const HEADLESS = process.env.HEADLESS || true;
+
+const LOGININFOS = {
+  user: {
```

Comment:
may be client rather than user

Reply:
yep but an admin is a super user :trollface:

Reply:
So it's a user :trollface:

Reply:
It's for making a difference between the admin in the BO and the user in the FO.